news/api.go

```
@@ -0,0 +1,244 @@
+			}
+		} else {
+			return nil, fmt.Errorf("wnAPI: %v", err)
+		}
+	}
+
+	if wn.bootstrap {
+		if err := wn.loadFromDumpFile(); err != nil {
+			err = fmt.Errorf("failed load from dump file: %v", err)
+			wn.logger.Debug(err)
+			wn.fetchTopNews(ctx)
+			wn.dump()
```

golangci-lint (1.62.2)

Error return value of `wn.dump` is not checked

(errcheck)